lib/node_modules/@stdlib/stats/array/nanrange-by/README.md

```diff
@@ -0,0 +1,157 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# nanrangeBy
+
+> Calculate the [range][range] of an array via a callback function, ignoring `NaN` values.
+
+<section class="intro">
+
```

Comment:
@gururaj1512 Would you mind adding the one line intro as in the other `array/*range*` packages?